http/swagger.yml

```diff
@@ -432,31 +434,31 @@ paths:
       operationId: PutTelegrafsID
       tags:
         - Telegrafs
-      summary: Update a telegraf config
+      summary: Update a Telegraf config
       parameters:
         - $ref: '#/components/parameters/TraceSpan'
         - in: path
           name: telegrafID
           schema:
             type: string
           required: true
-          description: ID of telegraf config
+          description: The Telegraf config ID.
```

Comment:
Some descriptions end with a period and some do not. Is there a pattern here I'm not seeing, or should we regularize? Not sure  :)

Reply:
I didn’t put periods on the response code descriptions and some of the request body descriptions. Theses didn’t seem appropriate for full sentences.